WORKSPACE

```diff
@@ -99,6 +99,17 @@ load("@build_bazel_rules_typescript//:package.bzl", "rules_typescript_dev_depend
 
 rules_typescript_dev_dependencies()
 
+# Install labs dependencies
+load("//packages/labs:package.bzl", "npm_bazel_labs_dependencies")
+
+npm_bazel_labs_dependencies()
+
+load("@rules_proto//proto:repositories.bzl", "rules_proto_dependencies", "rules_proto_toolchains")
+
+rules_proto_dependencies()
+
+rules_proto_toolchains()
+
```

Comment:
Was this re-ordering due to the rules_proto dep?

Reply:
yes, it needs to come earlier than rules_go to avoid getting their version in the maybe()